ui/core-react/src/core-react/messagebox/MessageBox.tsx

```diff
@@ -80,6 +81,7 @@ export class MessageBox extends React.PureComponent<MessageBoxProps> {
     modal: true,
   };
 
+  // eslint-disable-next-line deprecation/deprecation
   public override render(): JSX.Element {
```

Comment:
Can't we just drop the explicit types from the `render` of class components instead of ignoring the deprecation?